### ISSUE TITLE
UserWarning: Implicit dimension choice for softmax has been deprecated.

### ISSUE THREAD

**Issue by nikhilweee** [Ubiqus/OpenNMT-py#6]
I understand that this repository is still a work in progress, but I just wanted to point this out.  
```
onmt/modules/copy_generator.py:93: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  prob = F.softmax(logits)
```

**Comment by pltrdy** [comment]
Thanks for pointing this out.

Solved in: https://github.com/Ubiqus/OpenNMT-py/commit/04c67bd7eb920b1174655e3f963fb19a424773bc .
